app/models/post.rb

```diff
@@ -2,4 +2,12 @@ class Post < ApplicationRecord
   belongs_to :user, class_name: 'User', foreign_key: 'author_id', counter_cache: true
   has_many :comments, class_name: 'Comment', foreign_key: true
   has_many :likes, class_name: 'Like', foreign_key: true
+
```

Comment:
- [ ] Again, this needs to belong to `:author` according to the ERD.
- [ ] `:comments` and `:likes` do not need to point to a special class name, since their names are the same.
Look:

```ruby
belongs_to :author, class_name: 'User'
has_many :comments
has_many :likes
```